core/common/build.gradle

```diff
@@ -0,0 +1,51 @@
+plugins {
+    id 'com.android.library'
```

Comment:
All of our build.gradle files need this. I'll make a task for that.

Reply:
I've seen [spotless](https://github.com/diffplug/spotless) being used in a lot of our other repos. It seems like it could help with making sure our formatting is consistent and making sure we always have copyright headers.

Reply:
Tracking in b/282016086